### PR TITLE
Enabling secrets for ES certs and credentials

### PR DESCRIPTION
**Pull request by bhowd1**
Enabling secrets for ES certs and credentials

### CODE REVIEW

**Review by kingakowalska1 — CHANGES_REQUESTED** [review]
Hi Divyansh,

For the comments in the values.yaml file, please make the following changes:

Lines 64-66:

To specify a certificate using a secret, uncomment the esCertsSecret parameter and provide the secret name containing your certificate and certificate password (if any).
Use the full name of the certificate file (together with file extension, for example, “certificate.p12” or “certificate.jks”) as a key name in the secret. Use this key name to configure the “certificateName” parameter.
Use a key name “password” to provide the certificate password in the secret.


Lines 77-78:

To use a secret to configure basic authentication or TLS-based authentication between your external Elasticsearch service and SRS,
uncomment the esCredentialsSecret parameter and set it to the secret name. Use "username" and "password" as keys for your secret.

Thanks!

**Comment by bhowd1** [comment]
Thanks @kingakowalska1  for your comments, I've addressed them in the recent commit. Please check.

**Review by kingakowalska1 — APPROVED** [review]
Looks good! Thanks!

File: charts/backingservices/values.yaml [hunk 1]
```diff
@@ -61,6 +61,10 @@ srs:
     # Default certificatePassword value will be empty if not used.
     # certificateName: "Certificate_Name"
     # certificatePassword: "password"
+    # To specify a certificate using a secret, uncomment the esCertsSecret parameter and provide the secret name containing your certificate and certificate password (if any).
+    # Use the full name of the certificate file (together with file extension, for example, “certificate.p12” or “certificate.jks”) as a key name in the secret. Use this key name
+    # to configure the “certificateName” parameter. Use a key name “password” to provide the certificate password in the secret.
+    # esCertsSecret: srs-certificates
```
Comment (ravitejamekapega) [inline charts/backingservices/values.yaml:67]: "es" is redundant.. probably use `CertsSecret` name

**Review by ravitejamekapega — CHANGES_REQUESTED** [review]

File: charts/backingservices/values.yaml [hunk 2]
```diff
@@ -70,6 +74,9 @@ srs:
     # esCredentials:
     #   username: "username"
     #   password: "password"
+    # To use a secret to configure basic authentication or TLS-based authentication between your external Elasticsearch service and SRS,
+    # uncomment the esCredentialsSecret parameter and set it to the secret name. Use "username" and "password" as keys for your secret.
+    # esCredentialsSecret: srs-elastic-credentials
```
[the remark this answers is not in the document]
Comment (bhowd1) [reply charts/backingservices/values.yaml]: Addressed this in latest commit.

File: charts/backingservices/values.yaml (hunk 1 above)
Comment (bhowd1) [reply charts/backingservices/values.yaml]: Addressed this in latest commit

**Review by ravitejamekapega — APPROVED** [review]
Looks good !

**Pull request merged**